tasks/config/copy.js

```diff
@@ -12,6 +12,7 @@ module.exports = function (grunt) {
         '!src/fixtures/**',
         '!src/plugins/devMode/**',
         '!src/plugins/testsBundle/**',
+        '!src/plugins/console/public/tests/**',
```

Comment:
Should we just rename the `tests` directory to `__tests__` to be consistent with the rest of Kibana? That'd get picked up by the existing globs.